sdk/kfp-tekton-limitation.md

```diff
@@ -0,0 +1,36 @@
+
+# Limitations
+
+- [Overview](#overview)
+  - [Affinity, Node Selector, and Tolerations](#affinity-node-selector-tolerations)
+  - [Sidecars](#sidecars)
+  - [Store the *secrets* which need to pull images](#store-secrets-to-pull-images)
+
+## Overview
+
+During the development and testing, we noticed some limitations for the kfp-tekton project. Here we list those limitations. We are working on those limitations, some limitations are from the tekton pipeline projects, we have opened issues there to track the progress.
+
+<a name="affinity-node-selector-tolerations" />
+
+### Affinity, Node Selector, and Tolerations
+
+Please be aware that defined Affinity, Node Selector, and Tolerations are applied to all the tasks in the same pipeline because there's only one podTemplate allowed in each pipeline.
+
+<a name="sidecars" />
+
+### Sidecars
+
+When you run kfp-tekton pipeline with sidecars, you may notice a completed pod and a sidercar pod with Error. There are known issues with the existing implementation of sidecars:
```

Comment:
I redo a simple spelling test and found this last error.

```suggestion
When you run kfp-tekton pipeline with sidecars, you may notice a completed pod and a sidecar pod with errors. There are known issues with the existing implementation of sidecars:
```

Reply:
thanks Tommy. Fixed.